docs/notebooks/nb_05_examples.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>On the Root No. shouldn't there be 4 columns because it is ran in C2v symmetry?</p><pre class="ql-syntax" spellcheck="false">200 20 20     -0.89364609  #&lt;- 3 columns shouldn't there be 4 for A1, A2, B1, and B2?
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/evangelistalab/forte/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>